cuesubmit/cuesubmit/JobTypes.py

```diff
@@ -25,11 +25,13 @@ class JobTypes(object):
     SHELL = 'Shell'
     MAYA = 'Maya'
     NUKE = 'Nuke'
+    BLENDER = 'Blender'
 
     SETTINGS_MAP = {
         SHELL: SettingsWidgets.ShellSettings,
         MAYA: SettingsWidgets.BaseMayaSettings,
-        NUKE: SettingsWidgets.BaseNukeSettings
+        NUKE: SettingsWidgets.BaseNukeSettings,
+        BLENDER: SettingsWidgets.BaseBlenderSettings
```

Comment:
nit: You can end this line with a comma so any future additions don't also have to change this line.